src/main/java/frc/robot/subsystems/Climber_Subsystem.java

```diff
@@ -27,29 +28,100 @@
     private WPI_TalonSRX horizontalL = new WPI_TalonSRX(RobotMap.horizontalClimb_L);
     private AHRS ahrs = new AHRS(SPI.Port.kMXP);
     private double vp = 0.05;
-    private double hp = 0.8;
+    private double hp = 0;//0.8;
     private double vError, vResponce, hError, hResponce = 0;
 
+    private Boolean updateState = null; 
+
     private int oldLVertical, oldRVertical, oldLHorizontal, oldRHorizontal = 0;
 
     public Climber_Subsystem() {
         super();
-        verticalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
-        verticalL.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
-        horizontalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
-        horizontalL.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 10);
+        verticalR.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 50);
```

Comment:
No real change here, I increased the timeout on a hunch but I honestly don't think it matters.